src/main/java/com/griefcraft/lwc/LWC.java

```diff
@@ -928,7 +928,20 @@ public Player findPlayer(int minX, int maxX, int minY, int maxY, int minZ, int m
     }
 
     /**
-     * Check the data of locale
+     * Check the data of locale<br />
+     * Returns null if invalid<br />
+     *
+     * @param key key of locale
+     * @param args Character to be rewritten
+     *              Example: %block% --> Chest
+     * @return message or null
+     */
+    public String[] getLocaleMessage(String key, Object... args) {
```

Comment:
I don't know if this method is necessary, since you can just call the existing getLocaleMessage with the Console sender.

Reply:
The advantage is that the code is simplified.